docs/root/api-v2/bootstrap/bootstrap.rst

```diff
@@ -11,3 +11,4 @@ Bootstrap
   ../config/overload/v2alpha/overload.proto
   ../config/ratelimit/v2/rls.proto
   ../config/trace/v2/trace.proto
+  ../config/trace/v2alpha/xray.proto
```

Comment:
What I would recommend doing here in a follow up is just making a new directory/TOC for tracing, and then put these two RST in that with its own TOC. This will make the new docs look better. Later we can split out all the other tracers into their own files also. This is not a big deal so feel free to do this in your next change.